x-pack/plugins/data_visualizer/public/application/data_comparison/constants.ts

```diff
@@ -0,0 +1,1047 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const DATA_COMPARISON_TYPE = {
+  NUMERIC: 'numeric',
+  CATEGORICAL: 'categorical',
+  UNSUPPORTED: 'unsupported',
+} as const;
+
+export const NUMERIC_TYPE_LABEL = i18n.translate(
+  'xpack.dataVisualizer.dataComparison.numericLabel',
+  {
+    defaultMessage: 'Numeric',
+  }
+);
+export const CATEGORICAL_TYPE_LABEL = i18n.translate(
+  'xpack.dataVisualizer.dataComparison.categoricalLabel',
+  {
+    defaultMessage: 'Categorical',
+  }
+);
+
+export const UNSUPPORTED_LABEL = i18n.translate(
+  'xpack.dataVisualizer.dataComparison.UnsupportedLabel',
+  {
+    defaultMessage: 'Unsupported',
+  }
+);
+
+export const REFERENCE_LABEL = i18n.translate(
+  'xpack.dataVisualizer.dataComparison.referenceLabel',
+  {
+    defaultMessage: 'Reference',
+  }
+);
+
+export const COMPARISON_LABEL = i18n.translate(
+  'xpack.dataVisualizer.dataComparison.productionLabel',
+  {
+    defaultMessage: 'Comparison',
+  }
+);
+
+export const DATA_COMPARISON_TYPE_LABEL = {
+  [DATA_COMPARISON_TYPE.NUMERIC]: NUMERIC_TYPE_LABEL,
+  [DATA_COMPARISON_TYPE.CATEGORICAL]: CATEGORICAL_TYPE_LABEL,
+  [DATA_COMPARISON_TYPE.UNSUPPORTED]: UNSUPPORTED_LABEL,
+} as const;
+
+export const DRIFT_P_VALUE_THRESHOLD = 0.05;
+
+export const CRITICAL_VALUES_TABLE = [
```

Comment:
I'd like to suggest to add a comment here that includes the python code used to create these constants so we don't lose track of it.

Reply:
Updated here [`1b19e66` (#161365)](https://github.com/elastic/kibana/pull/161365/commits/1b19e6675d379d70a21e83e46b83d579f913217f)